### PR TITLE
[Quantum RPG] Nice kitty, good kitty

### PR DESCRIPTION
**Pull request by dstrain115**
- Add Schrodinger Cat as the final boss for the Hadamard Hills.

### CODE REVIEW

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py [hunk 1]
```diff
@@ -273,7 +275,16 @@
...
         title="Door of the Research Institute",
         description=("Room description forthcoming."),
-        encounters=[],
+        encounters=[
+            Encounter(
+                [
+                    SchrodingerCat("Guard Cat"),
+                ],
+                probability=1.0,
+                description="A cat both alie and dead hisses at you as you approach the door!",
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:284]: REPLACE alie
WITH alive

File: unitary/examples/quantum_rpg/npcs_test.py [hunk 2]
```diff
@@ -86,3 +86,23 @@ def test_purple_foam():
     assert msg == "PurpleFoam bubbles measures person_1 as HURT."
     msg = qar.act_on_enemy_qubit(c.get_hp("person_1"), 0.3, slime=0.25)
     assert msg == "PurpleFoam bubbles covers person_1 with foam!"
+
+
+def test_schrodinger_cat():
```
Comment (cognigami) [inline unitary/examples/quantum_rpg/npcs_test.py:91]: Would be nice to see a test of act_on_enemy_qubit().

**Review by cognigami — CHANGES_REQUESTED** [review]

File: unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py (hunk 1 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py:284]: Done.

File: unitary/examples/quantum_rpg/npcs_test.py (hunk 2 above)
Comment (dstrain115) [reply unitary/examples/quantum_rpg/npcs_test.py:91]: Done.  Also added a quantopedia entry now that the quantopedia is committed.

**Comment by dstrain115** [comment]
@cognigami Since you requested changes, it is blocking me from merging.  Can you re-review and approve when ready?  Thanks!

**Review by cognigami — APPROVED** [review]

**Pull request merged**